python/ray/data/block.py

```diff
@@ -53,7 +53,7 @@
 
 def _validate_key_fn(
```

Comment:
Also move this function to `SortKey`? It can be named as `validate_schema`. and we only  need to handle the `list[str]` case.

Reply:
Yep good idea - done.